### ISSUE TITLE
Documentation for attribute WordParagraph.underline links to MSDN

### ISSUE THREAD

**Issue by tmheath** [EvotecIT/OfficeIMO#173]
The documentation for this attribute found [here](https://github.com/EvotecIT/OfficeIMO/blob/master/Docs/officeimo.word.wordparagraph.md) links to [MSDN Nullable](https://learn.microsoft.com/en-us/dotnet/api/system.nullable-1?view=net-8.0) instead of UnderlineValues.
I'm having trouble finding the documentation that I need, I tried just passing a
string but apparently UnderlineValues is separate from string.

Where is the documentation for UnderlineValues? I can submit
a PR for modifying the docs, this probably should be done unless
I'm just being an idiot here.

**Comment by tmheath** [comment]
Alright gotcha... I'll be able to look into that tonight. Thanks, maybe just be able to generate links for each contained type.
Do you know offhand where the documentation is? I'll find it on my own eventually and it's perfectly fine if you don't.

**Comment by tmheath** [comment]
Thank you very much... I'm actually using your library to write a declarative library
that has sensible defaults for everything with the same level of control... I started
trying to use the OpenXML SDK but that has some kind of weird issue from Linux
at home so I found yours and that somehow works for that, not sure why. This
library seems excellent btw...

I've created an issue for the tool and pending they consider it a good idea and
I wouldn't have to fight through how the system is setup to access the document
link for the wrapped class then I will be following up with a PR hopefully inside of
a week either adding an option or making it default and either the highest nested
type or every type in the signature. [here](https://github.com/charlesdevandiere/xmldoc2md/issues/31)

I was a little worried initially as to whether or not I was being abrasive, sorry if I
came across that way.
